protocol/lib/big_math.go

```diff
@@ -264,3 +264,16 @@ func warmCache() map[uint64]*big.Int {
 
 	return bigExponentValues
 }
+
+// Int64MulPow10 returns the result of `val * 10^exponent`, in *big.Int.
+func Int64MulPow10(
```

Comment:
if this is only used in testing, I would prefer not introducing more very-specific functions into the `lib` folder

this generally creates more surface area for bugs which is a bad thing

if only for testing, is it okay to put it in `testutils`?

Reply:
Done